### ISSUE TITLE
Reset

### ISSUE THREAD

**Issue by Pedro1562** [Citillara/CitiBot#15]
The broadcaster should have a command to reset all time cookies.

**Comment by Citillara** [comment]
Well, I'm not sure if this should be a bot command but a website command. Pretty dangerous stuff

I'm also considering 2 types of reset.
1) Simply resets the current cookies each user are holding in the channel
2) Resets ALL cookie data associated to the channel (top cookie count and so on)

@Pedro1562 Can you comment on if you agree on those ?

**Comment by Pedro1562** [comment]
Yeah, those two types would be nice. It could be a bot command if the bot gave some warning before like."Careful, this will delete all cookie data saved for your channel" or something like that.

**Comment by Citillara** [comment]
Website only integration.
Simple reset will be backed up just once. Each new resets cancel the previous backup
Restore command on website

Full reset clears everything. No backup

Global counters not changed

**Comment by Citillara** [comment]
Should streamer be able to reset a single target ?
